tests/posix/spinlock/CMakeLists.txt

```
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(posix_spinlock)
+
+FILE(GLOB app_sources src/*.c)
+zephyr_include_directories(${ZEPHYR_BASE}/lib/posix)
```

These two lines can be removed
```suggestion
FILE(GLOB app_sources src/*.c)
zephyr_include_directories(${ZEPHYR_BASE}/lib/posix)
```